docs/without_helm.md

```diff
@@ -46,6 +46,17 @@ k8s-deploy-and-verify -f deploy/development.config
 
 More indepth examples are available in the [examples directory](https://github.com/reactiveops/rok8s-scripts/tree/master/examples).
 
+## Credentials
```

Comment:
This is an interesting choice.  It definitely works, but I think our preferred method is to generate a kubeconfig for a service account in the cluster.  I don't think we should encourage people to use their own personal kubeconfig for this (especially in the case where they use gcloud or aws-iam-authenticator since this won't work)

Reply:
That definitely makes sense. Is there a `kubectl` command I could replace `cat ~/.kube/config` with? I think we need some instructions for setting the `KUBECONFIG_DATA` env variable at least.

Reply:
I added a bunch of warnings about not doing this for serious deployments. But given how intense the service account process is, I think it's good to have a simple way to try out rok8s-scripts on a demo cluster. Objections?

Reply:
Seems fine for now.